<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registro</class>
 <widget class="QWidget" name="registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="texto_usuario">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="texto_empleado">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="etiqueta_usuario">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="texto_contrasenia">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="etiqueta_empleado">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Empleado :</string>
   </property>
  </widget>
  <widget class="QLabel" name="etiqueta_agregarempleado">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Empleado</string>
   </property>
  </widget>
  <widget class="QLabel" name="etiqueta_contrasenia">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrase√±a :</string>
   </property>
  </widget>
  <widget class="QLabel" name="etiqueta_eliminarempleado">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar empleado</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="boton_seleccion">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
